File: 程式語言 V2答案檔/C++/505.cpp
#include <iostream>
#include <cmath>
#include <iomanip>
using namespace std;

int main()
{
    double a, b, c, d, e, f, ans;
    cin >> a >> b >> c >> d >> e >> f;
    ans = abs(a) * floor(b) + pow(c, d) * sqrt(e) + log10(f);
    cout << setprecision(2) << setiosflags(ios::fixed);
    cout << ans;
    return (0);
}

File: 程式語言 V2答案檔/PHP/505.php
<?php
    $a = doubleval(fgets(STDIN));
    $b = doubleval(fgets(STDIN));
    $c = doubleval(fgets(STDIN));
    $d = doubleval(fgets(STDIN));
    $e = doubleval(fgets(STDIN));
    $f = doubleval(fgets(STDIN));
    printf("%.2f\n", abs($a) * floor($b) + pow($c, $d) * sqrt($e) + log10($f));
?>